runtests.py

```diff
@@ -0,0 +1,29 @@
+import unittest
```

Comment:
Addd more comments please. At the top, a """header describing what the code does with necessary inputs, optional inputs, and outputs written by XXX on YYY date """. Also add comments on lines periodically so we know what this is doing.